### PR TITLE
doc: update manual ansible playbook step

### LINKED ISSUE #21: doc | ibmcloud: Inconsistency in default values for `cos_service_instance_name `
From https://github.com/confidential-containers/cloud-api-adaptor/pull/18#discussion_r875737240:
> In the next section the doc says:
> > cos_service_instance_name is the COS Service Instance Name, optional, default to value based on your cluster_name > tfvar if not set, ${cluster_name}-cos-service-instance.
>
> I think we should make the defaults of these line up, otherwise it will get confusing.

We now have inconsistency in the default values of the `cos_service_instance_name` in the two sections it is used:
- In creating the COS instance we have:
  > cos_service_instance_name in variables.tf has the default value 
- In building the pod VM image and uploading it we have:
  > cos_service_instance_name is the COS Service Instance Name, optional, default to value based on your cluster_name tfvar if not set, ${cluster_name}-cos-service-instance.
  
This is not a good user experience, so we have a few choices:
- Either get rid of the defaults and stop them being optional altogether
- Change both defaults to `cos-image-instance`. I don't think there is a requirement for the cos service instance name to be unique across users, so this should work
- Change both defaults to `<cluster_name>-cos-image-instance`. Unless there is a way to read the `cluster_name` value from the cluster terraform.tfvars, I don't think this is a good solution, as there isn't another reason for cluster name to be required in the cos instance creation, so it's letting a user not supply one value by making them supply another
- Any other solution someone smarter than me can think of?

### PR DESCRIPTION
**Pull request by Jonah-F**
Updated README to reflect splitting of ansible playbook.
Step now runs two new playbooks consecutively.

Fixes #21

Signed-off-by: [Jonah-Farrow] <jonah.farrow@ibm.com>

### CODE REVIEW

File: ibmcloud/README.md [hunk 1]
```diff
@@ -136,7 +136,7 @@ This Terraform configuration also triggers execution of an Ansible playbook to s
 If ansible fails for some reason, you can rerun the ansible playbook as follows.
```
Comment (jtumber-ibm) [inline ibmcloud/README.md:136]: Small note but should this been updated to `ansible playbooks`?

**Review by jtumber-ibm — CHANGES_REQUESTED** [review]
Line 134, also has a link to the playbook which is no longer there.

File: ibmcloud/README.md (hunk 1 above)
Comment (Jonah-F) [reply ibmcloud/README.md:136]: Thanks, made both changes.

Comment (jtumber-ibm) [reply ibmcloud/README.md]: LGTM

**Review by jtumber-ibm — APPROVED** [review]

**Comment by Jonah-F** [comment]
**Commit message references wrong issue for fix**.
The commit `doc: update manual ansible playbook step` should reference issue 22, not issue 21, as to what the change fixes.

**Pull request merged**